specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2022-03-30-preview/examples/SqlDbCreateOrUpdateDatabaseMigrationMAX.json

```diff
@@ -0,0 +1,93 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "testrg",
+    "sqlDbInstanceName": "sqldbinstance",
+    "targetDbName": "db1",
+    "api-version": "2022-03-30-preview",
+    "parameters": {
+      "properties": {
+        "kind": "SqlDb",
+        "sourceDatabaseName": "aaa",
+        "scope": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/testrg/providers/Microsoft.Sql/servers/sqldbinstance",
+        "migrationService": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/testrg/providers/Microsoft.DataMigration/sqlMigrationServices/testagent",
+        "targetSqlConnection": {
+          "dataSource": "sqldbinstance",
+          "userName": "bbb",
+          "password": "placeholder",
+          "encryptConnection": true,
+          "trustServerCertificate": true,
+          "authentication": "SqlAuthentication"
+        },
+        "sourceSqlConnection": {
+          "dataSource": "aaa",
+          "userName": "bbb",
+          "password": "placeholder",
+          "encryptConnection": true,
+          "trustServerCertificate": true,
+          "authentication": "WindowsAuthentication"
+        },
+        "sqlDataCopyThresholds": {
+          "cidxRowThreshold": 10000,
+          "cidxKbsThreshold": 2000
+        },
+        "tableList": [
+          "[Schema1].[TableName1]",
+          "[Schema2].[TableName2]"
```

Comment:
The problem here is that this violates the GET - PUT parity for this resource. You PUT something and the GET returns something else. This is not how REST is supposed to work.

Reply:
We do return the tables that are being migrated, but that are part of property in response. We will add tableList as well to the response.

Reply:
done